otp.go

```diff
@@ -127,6 +127,46 @@ func (k *Key) Period() uint64 {
 	return 30 // If no period is defined 30 seconds is the default per (RFC 6238)
 }
 
+// Digits returns the digits value.
+func (k *Key) Digits() uint {
+	var u uint = 6 // Default value for referenced Google Authenticator implementation.
```

Comment:
I think we should return `0` as a default value. Google Authenticator is not the only app for OTP codes. Also, RFC 6238 doesn't mention defaults for digit param.

Reply:
done